docker/sklearn/verify.py

```diff
@@ -6,4 +6,9 @@
 import networkx
 from bs4 import BeautifulSoup
 
+import os
+import multiprocessing
+assert int(os.environ['OPENBLAS_NUM_THREADS']) == 1, f"{os.environ['OPENBLAS_NUM_THREADS']=}"
+assert int(os.environ['OMP_NUM_THREADS']) == multiprocessing.cpu_count(), f"{os.environ['OMP_NUM_THREADS']=}, {multiprocessing.cpu_count()=}"
```

Comment:
Would it be ok to `echo` the values participating in these `assert`s?
For debugging purposes.